docs/examples/table-v2/filter.vue

```diff
@@ -70,7 +70,7 @@ columns[0].headerCellRenderer = (props: HeaderCellSlotProps) => {
   return (
     <div class="flex items-center justify-center">
       <span class="mr-2 text-xs">{props.column.title}</span>
-      <ElPopover ref={popoverRef} trigger="click">
+      <ElPopover ref={popoverRef} trigger="click" width="180">
```

Comment:
## Suggest

```
<ElPopover
  ref={popoverRef}
  trigger="click"
  :width="180" // here
>
```

[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtcG9wb3ZlclxuICAgIHBsYWNlbWVudD1cImJvdHRvbVwiXG4gICAgdGl0bGU9XCJUaXRsZVwiXG4gICAgOndpZHRoPVwiMjAwXCJcbiAgICB0cmlnZ2VyPVwiY2xpY2tcIlxuICAgIGNvbnRlbnQ9XCI6d2lkdGg9JzIwMCdcIlxuICA+XG4gICAgPHRlbXBsYXRlICNyZWZlcmVuY2U+XG4gICAgICA8ZWwtYnV0dG9uPkNsaWNrIHRvIGFjdGl2YXRlPC9lbC1idXR0b24+XG4gICAgPC90ZW1wbGF0ZT5cbiAgPC9lbC1wb3BvdmVyPlxuICAgPGVsLXBvcG92ZXJcbiAgICBwbGFjZW1lbnQ9XCJib3R0b21cIlxuICAgIHRpdGxlPVwiVGl0bGVcIlxuICAgIHdpZHRoPVwiMjAwXCJcbiAgICB0cmlnZ2VyPVwiY2xpY2tcIlxuICAgIGNvbnRlbnQ9XCJ3aWR0aD0nMjAwJ1wiXG4gID5cbiAgICA8dGVtcGxhdGUgI3JlZmVyZW5jZT5cbiAgICAgIDxlbC1idXR0b24+Q2xpY2sgdG8gYWN0aXZhdGU8L2VsLWJ1dHRvbj5cbiAgICA8L3RlbXBsYXRlPlxuICA8L2VsLXBvcG92ZXI+XG48L3RlbXBsYXRlPlxuXG48c2NyaXB0IGxhbmc9XCJ0c1wiIHNldHVwPlxuaW1wb3J0IHsgcmVmIH0gZnJvbSAndnVlJ1xuXG5jb25zdCB2aXNpYmxlID0gcmVmKGZhbHNlKVxuPC9zY3JpcHQ+XG4iLCJpbXBvcnRfbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7fVxufSIsIl9vIjp7fX0=)

If you look at the example, you can see the difference between `:width` and `width`. It seems that you need to use `:width` to enter the size normally.

Reply:
yeah, you are right. I should use v-bind to bind props value for width, but `:width` don't work here becuase of tsx.
so I use `{...{width: 200}}` to replace `:width`.
thanks your suggest!